libraries/rush-lib/src/logic/pnpm/SplitWorkspaceGlobalPnpmfileShim.ts

```diff
@@ -9,6 +9,8 @@
 
 import path from 'path';
 
+import normalizePath from 'normalize-path';
```

Comment:
Actually, we can not import third party package by "import" in this file, we need to patch the module to "context" which is initialized in "init" function. You can see the related logic for patching "semver" module to "context"